app/assets/images/svg/vendor-ibm_power_vc_storage.svg

```diff
@@ -0,0 +1 @@
+vendor-ibm_power_vc.svg
```

Comment:
Should this one be `ibm_power_vc_swift` ?

Reply:
Yeah, that's correct since there is no generic storage manager. I don't think PowerVC even supports swift object storage, but we do have the swift storage classes defined.